src/Facility.Definition/Fsd/FsdParsers.cs

```diff
@@ -137,6 +137,34 @@ internal static class FsdParsers
 			context.GetPart(ServicePartKind.Keyword, keyword),
 			context.GetPart(ServicePartKind.Name, name));
 
+	private static IParser<ServiceExternalDtoInfo> ExternalDtoParser(Context context) =>
+		from comments1 in CommentOrWhiteSpaceParser.Many()
+		from attributes in AttributeParser(context).Delimited(",").Bracketed("[", "]").Many()
+		from comments2 in CommentOrWhiteSpaceParser.Many()
+		from keyword in KeywordParser("externdata")
```

Comment:
i think `extern data` is nicer than `externdata` but didn't investigate

Reply:
I agree.